en/admin/index.md

```diff
@@ -10,23 +10,26 @@ nav_order: 2
 # Administrator's guide
 
 Welcome to the administrator's guide for CoopCycle platform.
-{: .fs-6 }
```

Comment:
Shouldn't we keep the `{: .fs-6}` utility here? [see docs](https://pmarsceill.github.io/just-the-docs/docs/utilities/typography/)

Reply:
It didn't actually make any difference as it wasn't on any other pages. Shouldn't this be set in the CSS rather than the markdown itself?

Reply:
If it doesn't make any difference, ok 🙂 

We are using what is shipped with "Just The Docs" theme. There shouldn't be the need to touch the CSS. 

I don't remember, it was @elbuffeto, she did all the work (see #19)